algosdk/dryrun_results.py

```diff
@@ -0,0 +1,228 @@
+from typing import List
+
+import tabulate
+
+tabulate.MIN_PADDING = 0
+
+from tabulate import tabulate, TableFormat, DataRow
+import base64
```

Comment:
```suggestion
import base64
from typing import List

import tabulate
from tabulate import tabulate, TableFormat, DataRow


tabulate.MIN_PADDING = 0
```

Reply:
I dont think i can do it like that, It has to be import tabulate then change the global. The thing i import from tabulate is also called tabulate